exercises/binary-search/tests/binary-search.rs

```diff
@@ -0,0 +1,97 @@
+extern crate binary_search;
+
+use binary_search::find;
+
+#[test]
+fn finds_a_value_in_an_array_with_one_element() {
+    assert_eq!(find(&[6], 6), Some(0));
+}
+
+#[test]
+#[ignore]
+fn finds_first_value_in_an_array_with_two_element() {
+    assert_eq!(find(&[1, 2], 1), Some(0));
```

Comment:
note, I don't see these in https://github.com/exercism/problem-specifications/blob/master/exercises/binary-search/canonical-data.json , that's fine, but did you add them yourself?

Should they be added to https://github.com/exercism/problem-specifications/blob/master/exercises/binary-search/canonical-data.json ?

Reply:
I have added them by myself because in the canonical section there is a big step between the 1-element test case and the 7-element test case. For incremental (TDD-wise)  coding it's easier to get it first working with an 2-element array (no loops or recursion needed). But maybe this is just my personal thinking style, so I don't know if it should go into the canonical test data.

Reply:
> there is a big step between the 1-element test case and the 7-element test case

That sounds like something that all language tracks can and should benefit from. It is in line with the sentence "manageable increase in complexity" that is noted in https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/test-suites.md. Therefore it should be offered as a PR to problem-specifications. I wouldn't want the Rust track to look selfish (whether or not we actually are selfish)

Reply:
OK, I will make a PR in the next days.